source/reference/beta-program/orr.txt

```diff
@@ -13,3 +13,54 @@ Enable Oplog Rollover Resilience
    :class: singlecol
 
 .. include:: /includes/beta-feature.rst
+
+Starting in {+c2c-beta-program-short+} 1.8, you can enable Oplog Rollover 
+Resilience (ORR). With ORR,  ``mongosync`` applies changes made on the source 
+cluster to the destination cluster during the initial sync.
+
+By applying changes earlier in the sync process, ``mongosync`` maintains a more 
+recent position in the :term:`oplog`. This adds resilience to long-running 
+operations, mitigates the risk of oplog rollover, and significantly lowers 
+the risk of restarting the sync.
+
+Syntax 
+------
+
+To enable ORR, use ``--oplogRolloverResilienceIntervalSeconds``and specify the 
+interval, in seconds, in which ``mongosync`` checks for eligible change events 
+in the oplog. The default value is ``-1``, which disables the option.
```

Comment:
```suggestion
in the oplog. The default value is ``-1``, which disables ORR.
```
An option can't be disabled per se; it's the ORR mechanism that gets disabled.

Reply:
Noted and updating!